Main/lib/xml/workflow/genesAndProteins.xml

```diff
@@ -194,11 +195,14 @@
       <depends name="runLowComplexityFilter"/>
     </step>
 
-
     <subgraph name="interpro" xmlFile="interpro.xml" includeIf="$$isNotEbiGenome$$">

```

Comment:
I was thinking we may want to remove the "includeIf" bit here and always run the interproscan, signalp and tmhmm steps.  
That is more work for data loaders up front but simpler and cleaner moving forward

Reply:
Ok. I will remove the include if here for interproscan.